src/tests/suites/RedactUtility-Test.ts

```diff
@@ -42,7 +43,7 @@ suite('Redact Test', () => {
         // Now lets test the Invalid Json Erro when incorrect json is fed into the funct.
         const result = redactJsonObject(error);
         const containsInvalidJson = result.message.toString().includes("Invalid Json");
-        assert.equal(containsInvalidJson, true);
+        expect(containsInvalidJson).to.equal(true);
```

Comment:
💡 idea / suggestion: please note and if there is a reading for use of riwil use here, instead of using -   ‘ expect(true).to.be.true;’  https://www.chaijs.com/api/bdd/#true

Reply:
done, updated.